layouts/community/ortho_4x12/mindsound/keymap.c

```diff
@@ -0,0 +1,327 @@
+// This is the canonical layout file for the Quantum project. If you want to add another keyboard,
+// this is the style you want to emulate.
+
+#include QMK_KEYBOARD_H
+
+#include "action_layer.h"
```

Comment:
Most of these includes aren't needed.  The clicky, and flicker ones are probably the only ones needed here.

Reply:
Hm, I needed QMK_KEYBOARD_H or my branch wouldn't compile.

Reply:
Yes, that's the one that is needed. :)
The rest most likely not.